prymer/primer3/primer3_parameters.py

```diff
@@ -186,6 +188,17 @@ class ProbeParameters:
 
     Please see the Primer3 manual for additional details: https://primer3.org/manual.html#globalTags
 
+    Primer3 uses both thermodynamic and alignment-based approaches to quantify oligo
+    self-complementarity. In general, these settings are meant to limit problematic oligo
+    self-complementarity and avoid primer-dimers or other nonspecific binding of probes.
```

Comment:
I think I would re-word this to say something like "While Primer3 supports alignment based and thermodynamic methods for simulating hybridizations, prymer enforces the use of the thermodynamic approach which is slightly more computationally expensive but superior."